### ISSUE TITLE
Fix consistency in paper

### ISSUE THREAD

**Issue by florinbarbisch** [Satellite-Based-Crop-Classification/messis#83]
- [x] Section/Appendix/Figure/Table 5.1 => grossgeschrieben
- [x] ZueriCrop/ZüriCrop => ZueriCrop
- [x] pre-training/pretraining => pretraining
- [x] timesteps/time-steps => timesteps
- [x] F1 Score or F1 score or F1-score => F1 score
- [x] Accuracy/accuracy => accuracy
- [x] Alle metriken klein schreiben ausser titel
  - [x] recall
  - [x] precision
  - [x] kappa
- [x] Almost always use \ref in combination with \nameref
   - [x] search and check for all \ref
   - [x] search and check for all \nameref
- [x] our study/thesis/work => our thesis/work (be careful not to mess up references)
- [x] pixel-wise/field-majority => pixel-wise/field majority
- [x] deep learning/deep-learning => deep learning
- [x] per class/per-class => per-class
- [x] {tab:metrics_comparison_exp-10-fcn-head-kernel-size}: Could also refer to experiment section with refname (maybe we should do that, consistently) "the experiment in Secction \ref{} \namref{}" => Experiment \ref \nameref
  - [x] search for "experiment"
  - [x] rename labels from sec:___ to exp:___ in experiment section
  - [x] maybe search for exp-\d
- [x] replace references to tables in results with Experiment \ref \nameref
  - [x] search for \ref{tab:metrics_compariso

**Comment by florinbarbisch** [comment]
Done!
